atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -117,6 +118,7 @@
     Session session, longRunningQuerySession;
 
     CQLStatementCache cqlStatementCache;
+    CQLKeyValueServices cqlKeyValueServices;
```

Comment:
Yep, happy to change all of them. Will probably delete this branch and use the first one though. And I still need to figure out a way to unstuck travis.